lib/happo/public/happo-runner.js

```diff
@@ -162,6 +162,14 @@ window.happo = {
     box.right = Math.max(box.right, rect.right);
     box.top = Math.min(box.top, rect.top);
 
+    if (box.left < 0) {
+      box.left = 0; // Don't go outside window
```

Comment:
It would be better to do this at the very end, next to the width and height calculations, instead of doing this for every node.

Also, you can use `Math.max` if you like.